src/Apps/Order/Components/ExpressCheckout/ExpressCheckoutUI.tsx

```diff
@@ -329,6 +329,22 @@ export const ExpressCheckoutUI = ({ order }: ExpressCheckoutUIProps) => {
       const { error, confirmationToken } = await stripe.createConfirmationToken(
         {
           elements,
+          params:
+            shippingRate?.id === "PICKUP"
```

Comment:
👍  Let's add a test case for this! It can be added [here](src/Apps/Order/Components/ExpressCheckout/__tests__/ExpressCheckoutUI.jest.tsx). I'm not an expert but happy to pair!